platform/darwin/src/MGLGeometry_Private.h

```diff
@@ -6,6 +6,7 @@
 #endif
 
 #import <mbgl/util/geo.hpp>
+#import <mbgl/util/geometry.hpp>
```

Comment:
Alternatively, you can put `MGLPointFromLocationCoordinate2D()` in MGLFeature_Private.h, which already houses some geometry type conversion code.